### PR TITLE
[KOGITO-1654] - Guide for Core/Runtimes team to Smoke test local changes on Kubernetes

### PR DESCRIPTION
**Pull request by Kaitou786**
See: https://issues.redhat.com/browse/KOGITO-1654

Many thanks for submiting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributors guide](CONTRIBUTING.MD#sending-a-pull-request)
- [x] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster

### CODE REVIEW

**Comment by Kaitou786** [comment]
See the file here: https://github.com/Kaitou786/kogito-cloud-operator/blob/KOGITO-1654/GUIDE_TO_KOGITO_DEVS.md

File: GUIDE_TO_KOGITO_DEVS.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+As an example for building the Image for your Kogito custom Service we’re going to build an example from [kogito-examples](https://github.com/kiegroup/kogito-examples) of `process-business-rules-quarkus`
+
+```
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
+```
+Now create a Dockerfile inside the `kogito-examples/process-business-rules-quarkus`
+
+With the following content:
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:85]:
```suggestion
Now create a `Dockerfile` inside the `kogito-examples/process-business-rules-quarkus` with the following content:
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+```
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:87]:
```suggestion
```Dockerfile
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 3]
```diff
@@ -0,0 +1,162 @@
...
+COPY target/*-runner.jar $KOGITO_HOME/bin    
+COPY target/lib $KOGITO_HOME/bin/lib
+```
+
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:99]:
```suggestion

```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 4]
```diff
@@ -0,0 +1,162 @@
...
+Now let’s push the above built image to a public container registry.
+
+`$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest`
+
+## Install the Operator
+First we would need to enable the OLM in our Minikube cluster for that just run:
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:105]:
```suggestion
First we would need to enable the OLM in our Minikube cluster, for that just run:
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+
+`$ minikube addons enable olm`
+
+Launch the OLM console locally, 
+Just clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repo and from the root of the project run: `$ make run-console-local`.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:110]:
```suggestion
To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+ 
+ This will run the operatorhub console on http://localhost:9000 
+
+Note: Needs to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:114]:
```suggestion
**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 7]
```diff
@@ -0,0 +1,162 @@
...
+
+Create a different namespace where kogito-operator and all the dependent operator will run
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:116]:
```suggestion
Create a different namespace where `kogito-operator` and all the dependent operator will run
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 8]
```diff
@@ -0,0 +1,162 @@
...
+
+`$ kubectl create ns kogito`
+
+Now on your browser and visit  https://localhost:9000. Select `Operators > OperatorHub` and search for kogito.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:120]:
```suggestion
Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`. 
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 9]
```diff
@@ -0,0 +1,162 @@
...
+ This will run the operatorhub console on http://localhost:9000 
+
+Note: Needs to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where kogito-operator and all the dependent operator will run
+
+`$ kubectl create ns kogito`
+
+Now on your browser and visit  https://localhost:9000. Select `Operators > OperatorHub` and search for kogito. 
+Select the kogito operator by Redhat and install it with defaults only changing the namespace where it needs to be installed. You can select the `kogito` namespace which was created for this purpose.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:121]:
```suggestion
Select the Kogito Operator by Red Hat and install it with defaults only changing the namespace where it needs to be installed. You can select the `kogito` namespace which was created for this purpose.
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 10]
```diff
@@ -0,0 +1,162 @@
...
+
+You can see the pods by:
+
+`$ kubectl get po -n kogito`
+
+Wait till all pods are in running state (It is installing kogito-operator and all the dependent operator for kogito)
+
+
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:129]:
```suggestion

```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 11]
```diff
@@ -0,0 +1,162 @@
...
+Alternatively you if you are having trouble in having the operators shown in the OperatorHub.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:130]: I'm not sure if I understood this sentence. Could you rephrase?

File: GUIDE_TO_KOGITO_DEVS.md [hunk 12]
```diff
@@ -0,0 +1,162 @@
...
+You can also install it from [here](https://operatorhub.io/operator/kogito-operator)
+
+## Run Kogito Service with Kogito Operator
+
+Now we’ll create a `KogitoRuntime` object with the Application image that we built earlier.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:135]:
```suggestion
Now we’ll create a `KogitoRuntime` object with the image we built and pushed earlier.
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 13]
```diff
@@ -0,0 +1,162 @@
...
+
+```
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:137]:
```suggestion
```yaml
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 14]
```diff
@@ -0,0 +1,162 @@
...
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoRuntime
+metadata:
+  name: process-business-rules-quarkus
+spec:
+  replicas: 1
+  image:
+    namespace: <your-quay-namespace>
+```
+Create a myapp.yml with the above content.
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:147]:
```suggestion
Create a `myapp.yml` file with the above content.
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 15]
```diff
@@ -0,0 +1,162 @@
...
+You can see your runtime object by 
+
+`$ kubectl get KogitoRuntime -n <operator-namespace>`
+
+Now expose the service on NodePort so it’s easily accessible.
+
+`$ kubectl expose deployment process-business-rules-quarkus -n <operator-namespace>  --type=NodePort --name=process-business-rules-quarkus`
+
+`$ minikube service process-business-rules-quarkus -n <operator-namspace>`
+
```
Comment (ricardozanini) [inline GUIDE_TO_KOGITO_DEVS.md:162]: Maybe adding the output, so users will see the location of the exposed service?

**Comment by ricardozanini** [comment]
I've added more people to take a look since we are aiming different personas to understand how to run a smoke test locally to avoid merging changes that could impact the operator.

File: GUIDE_TO_KOGITO_DEVS.md (hunk 15 above)
Comment (Kaitou786) [reply GUIDE_TO_KOGITO_DEVS.md:162]: The above command will open the service in your default browser, do you want to me add a screenshot with that?

**Comment by radtriste** [comment]
could we have that file in a `docs` folder ?

File: GUIDE_TO_KOGITO_DEVS.md [hunk 16]
```diff
@@ -0,0 +1,168 @@
...
+Now just build your image. Inside the above created directory run:
+
+`$ podman build -t <prefered-tag-name>  -f jobs-service-Dockerfile  .`
+
+The above command will build your image with `<prefered-tag-name>` and can be seen in the output of `$ podman images`
+
+**Note**: We added the `-f` flag to specify the `Dockerfile` in case you want to build multiple images, you can then have different files and specify them with the `-f` flag.
+This image needs to be on a public container registry from which it can be pulled later.
+```shell-script
+$ podman tag <prefered-tag-name> quay.io/<your-namespace>/<image-name>
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:76]: why not calling it directly `quay.io/<your-namespace>/<image-name>` in the previous command ?

File: GUIDE_TO_KOGITO_DEVS.md [hunk 17]
```diff
@@ -0,0 +1,168 @@
...
+Eg: podman tag jobs-service quay.io/myorg/jobs-service:latest
+$ podman push quay.io/<your-namespace>/<image-name>
+```
+
+## Build the Kogito Service Image
+
+ ### Base Images
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:83]:
```suggestion
### Base Images

```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 18]
```diff
@@ -0,0 +1,168 @@
...
+  * kogito-quarkus-ubi8
+  * kogito-quarkus-jvm-ubi8
+  * kogito-springboot-ubi8
+
+As an example for building the Image for your Kogito custom Service we’re going to build the example  `process-business-rules-quarkus` from [kogito-examples](https://github.com/kiegroup/kogito-examples) repository.
+
+```shell-script
+cd kogito-examples/process-business-rules-quarkus
+mvn clean package
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:92]:
```suggestion
$ git clone https://github.com/kiegroup/kogito-examples
$ cd kogito-examples/process-business-rules-quarkus
$ mvn clean package
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 19]
```diff
@@ -0,0 +1,168 @@
...
+```shell-script
+$ podman build --tag quay.io/<yournamespace>/process-business-rules-quarkus:latest .
+```
+It’ll build your image with the kogito service running on it.
+
+Now let’s push the above built image to a public container registry.
+
+`$ podman push quay.io/<yournamespace>/process-business-rules-quarkus:latest`
+
+## Install the Operator
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:112]:
```suggestion
## Install the Operator

```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 20]
```diff
@@ -0,0 +1,168 @@
...
+First we would need to enable the OLM in our Minikube cluster, for that just run:
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:113]:
```suggestion
First we would need to enable the OLM in our Minikube cluster. For that, just run:
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 21]
```diff
@@ -0,0 +1,168 @@
...
+
+`$ minikube addons enable olm`
+
+To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
+ 
+This will run the operatorhub console on http://localhost:9000 
+
+**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where `kogito-operator` and all the dependent operator will run
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:123]:
```suggestion
Create a different namespace where `kogito-operator` and all the dependent operator(s) will run
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 22]
```diff
@@ -0,0 +1,168 @@
...
+
+`$ kubectl create ns kogito`
+
+Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:127]:
```suggestion
Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.  
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 23]
```diff
@@ -0,0 +1,168 @@
...
+This will run the operatorhub console on http://localhost:9000 
+
+**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where `kogito-operator` and all the dependent operator will run
+
+`$ kubectl create ns kogito`
+
+Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`. 
+Select the Kogito Operator by Red Hat and install it with defaults only changing the namespace where it needs to be installed. You can select the `kogito` namespace which was created for this purpose.
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:128]: `and install it with defaults only changing the namespace where it needs to be installed`
can you rephrase ?

File: GUIDE_TO_KOGITO_DEVS.md [hunk 24]
```diff
@@ -0,0 +1,168 @@
...
+
+You can see the pods by:
+
+`$ kubectl get po -n kogito`
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:132]:
```suggestion
`$ watch kubectl get pod -n kogito`
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 25]
```diff
@@ -0,0 +1,168 @@
...
+
+Wait till all pods are in running state (It is installing kogito-operator and all the dependent operator for kogito)
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:134]:
```suggestion
Wait until all pods are in running state (It is installing kogito-operator and all the dependent operator(s) for kogito)
```

File: GUIDE_TO_KOGITO_DEVS.md [hunk 26]
```diff
@@ -0,0 +1,168 @@
...
+
+```yaml
+apiVersion: app.kiegroup.org/v1alpha1
+kind: KogitoRuntime
+metadata:
+  name: process-business-rules-quarkus
+spec:
+  replicas: 1
+  image:
+    namespace: <your-quay-namespace>
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:150]: please provide full image description, with registry, name and tag

File: GUIDE_TO_KOGITO_DEVS.md [hunk 27]
```diff
@@ -0,0 +1,168 @@
...
+```
+Create a `myapp.yml` file with the above content.
+
+Just create the above object with
+
+`$ kubectl create -f myapp.yml -n <operator-namespace>`
```
Comment (radtriste) [inline GUIDE_TO_KOGITO_DEVS.md:156]: why using `operator-namespace` here whereas you used `kogito` in previous part ?

File: GUIDE_TO_KOGITO_DEVS.md [hunk 28]
```diff
@@ -0,0 +1,168 @@
...
+
+This will deploy your artifacts in `~/.m2/repository/org/kie/kogito`. You can just build the artifacts you modified or added new feature in and update them in the Images
+
+## Build the Kogito Application Images
+We’re going to use our pre-built images which are available in the `quay.io/kiegroup/` namespace.
+
+ ### Images list
+  * kogito-data-index
+  * kogito-jobs-service
+  * kogito-management-console
```
[the remark this answers is not in the document]
Comment (Kaitou786) [reply GUIDE_TO_KOGITO_DEVS.md:46]: These are the application images of kogito, they will be used for updating with custom application jar if modified.

File: GUIDE_TO_KOGITO_DEVS.md (hunk 15 above)
Comment (ricardozanini) [reply GUIDE_TO_KOGITO_DEVS.md:173]: No need! Yes, you're right! :)

File: GUIDE_TO_KOGITO_DEVS.md [hunk 29]
```diff
@@ -0,0 +1,167 @@
...
+* [Run Kogito Service with Kogito Operator](#run-kogito-service-with-kogito-operator)
+
+## Install Minikube
+
+We decided to go with Minikube as the Kubernetes cluster as it is very resource efficient and can be started easily on any system.
+ ### Prerequisites
+  * Install `kubectl` binaries on your system [see](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+  * Have a hypervisor installed (kvm  is recommended for linux)
+
+Alternatively, you can install it using podman or docker driver which will launch the minikube cluster inside the containers.
```
[the remark this answers is not in the document]
Comment (Kaitou786) [reply GUIDE_TO_KOGITO_DEVS.md]: Removed the line the description is written on the kubernetes docs link which I already have for installing the minikube

File: GUIDE_TO_KOGITO_DEVS.md (hunk 28 above)
[the remark this answers is not in the document]
Comment (radtriste) [reply GUIDE_TO_KOGITO_DEVS.md]: but we are not going to use them directly. We are going to build new ones or ?

[the remark this answers is not in the document]
Comment (Kaitou786) [reply GUIDE_TO_KOGITO_DEVS.md]: We are using them as base image, and update the jar artifacts, building from scratch is too time consuming.

File: GUIDE_TO_KOGITO_DEVS.md [hunk 30]
```diff
@@ -0,0 +1,174 @@
...
+
+`$ kubectl create -f myapp.yml -n kogito`
+
+You can see your runtime object by 
+
+`$ kubectl get KogitoRuntime -n kogito`
+
+Now expose the service on NodePort so it’s easily accessible.
+
+`$ kubectl expose deployment process-business-rules-quarkus -n <operator-namespace>  --type=NodePort --name=process-business-rules-quarkus`
```
Comment (spolti) [inline GUIDE_TO_KOGITO_DEVS.md:170]: also need to mention that the ingress addon should be enabled, I don't think it is enabled by default.

File: docs/GUIDE_TO_KOGITO_DEVS.md [hunk 31]
```diff
@@ -0,0 +1,174 @@
...
+First, create a directory e.g: `~/images/`.
+Then, inside this directory, copy the artifacts you need to update:
+
+```shell-script
+$ cp /path/to/kogito-apps/jobs-service/target/jobs-service-8.0.0-SNAPSHOT-runner.jar .
+```
+
+Also, inside this directory, you can have a `Dockerfile` which can be used to update the artifact.
+
+` $ vim jobs-service-Dockerfile`
```
Comment (radtriste) [inline docs/GUIDE_TO_KOGITO_DEVS.md:61]: please use the same structure for all shell-script, even if it is only one line

File: docs/GUIDE_TO_KOGITO_DEVS.md [hunk 32]
```diff
@@ -0,0 +1,174 @@
...
+
+You can see your runtime object by 
+
+`$ kubectl get KogitoRuntime -n kogito`
+
+Now expose the service on NodePort so it’s easily accessible.
+
+`$ kubectl expose deployment process-business-rules-quarkus -n <operator-namespace>  --type=NodePort --name=process-business-rules-quarkus`
+
+`$ minikube service process-business-rules-quarkus -n <operator-namspace>`
```
Comment (radtriste) [inline docs/GUIDE_TO_KOGITO_DEVS.md:172]: put togeter

File: docs/GUIDE_TO_KOGITO_DEVS.md [hunk 33]
```diff
@@ -0,0 +1,174 @@
...
+
+Before proceeding, please read the README files on the following repositories to know the dependencies you might need to install to build and deploy the artifacts.
+ * [kogito-runtimes](https://github.com/kiegroup/kogito-runtimes)
+ * [kogito-apps](https://github.com/kiegroup/kogito-apps)
+ * [kogito-examples](https://github.com/kiegroup/kogito-examples)
+ 
+To generate the jar artifacts, you can just run the following command on each repository:
+
+```shell-script
+mvn clean package -DskipTests #(For skipping tests)
```
Comment (danielezonca) [inline docs/GUIDE_TO_KOGITO_DEVS.md:36]: Can you please add `-DskipTestsIT`? We bound them for now to `verify` phase so it is not really needed but it is safer :)

File: docs/GUIDE_TO_KOGITO_DEVS.md [hunk 34]
```diff
@@ -0,0 +1,190 @@
...
+
+Wait until all pods are in running state (It is installing kogito-operator and all the dependent operator(s) for kogito)
+
+Alternatively, you can install the `Kogito Operator` from [here](https://operatorhub.io/operator/kogito-operator)
+
+## Run Kogito Service with Kogito Operator
+
+Now we’ll create a `KogitoRuntime` object with the image we built and pushed earlier.
+
+```yaml
```
Comment (vaibhavjainwiz) [inline docs/GUIDE_TO_KOGITO_DEVS.md:157]: User can also deploy generated image using `kogito-cli` using below cmd :
`kogito deploy-service process-business-rules-quarkus --image quay.io/<your-quay-namespace>/process-business-rules-quarkus:latest`

File: docs/GUIDE_TO_KOGITO_DEVS.md (hunk 33 above)
Comment (Kaitou786) [reply docs/GUIDE_TO_KOGITO_DEVS.md]: Added

File: docs/GUIDE_TO_KOGITO_DEVS.md [hunk 35]
```diff
@@ -0,0 +1,190 @@
...
+
+## Install the Operator
+
+First we would need to enable the OLM in our Minikube cluster. For that, just run:
+
+```shell-script
+$ minikube addons enable olm
+```
+
+To launch the OLM console locally, clone the [operator-lifecycle-manager](https://github.com/operator-framework/operator-lifecycle-manager) repository and from the root of the project run: `$ make run-console-local`.
```
Comment (evacchi) [inline docs/GUIDE_TO_KOGITO_DEVS.md:128]: do I need go for this to work? I have cloned and run the `make run-console-local` successfully, although it did complain about go.

File: docs/GUIDE_TO_KOGITO_DEVS.md [hunk 36]
```diff
@@ -0,0 +1,190 @@
...
+
+**Note**: You will need to have [`jq`](https://stedolan.github.io/jq/manual/) installed and 9000 port available on the system.
+
+Create a different namespace where `kogito-operator` and all the dependent operator(s) will run
+
+```shell-script
+$ kubectl create ns kogito
+```
+
+Now open your browser and visit the OLM console on https://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.
```
Comment (evacchi) [inline docs/GUIDE_TO_KOGITO_DEVS.md:140]:
```suggestion
Now open your browser and visit the OLM console on http://localhost:9000. Select `Operators > OperatorHub` and search for `kogito`.  
```

http not https, otherwise it won't open.

I have followed all instructions up to here successfully, but I could not find or search for kogito. This is what I get:

> ### No OperatorHub Items Found
> Please check that the OperatorHub is running and that you have created a valid OperatorSource. For more information about OperatorHub, please click here.

image

I see that I need to set an OperatorSource. Which one?

Comment (Kaitou786) [reply docs/GUIDE_TO_KOGITO_DEVS.md:140]: I noticied it happens with sometimes, not really sure why. I'll check this out. Can you try the further steps with the alternate way to install the operator, Note: that installs the operator in `my-kogito-operator` namespace so use that namespace with rest of the commands (with `-n` flag).

File: docs/GUIDE_TO_KOGITO_DEVS.md (hunk 35 above)
Comment (Kaitou786) [reply docs/GUIDE_TO_KOGITO_DEVS.md:128]: Ah yes, the project requires the `go1.12+` to be installed, I'll update that in per-requisites.

File: docs/GUIDE_TO_KOGITO_DEVS.md (hunk 36 above)
Comment (ricardozanini) [reply docs/GUIDE_TO_KOGITO_DEVS.md:140]: I believe that takes some time for the OLM to index all the manifests out there.

**Comment by Kaitou786** [comment]
> @Kaitou786 and I went through it together and sorted out my issues, so as long as those are documented, this is good to go!

Documented the information in this [commit](https://github.com/kiegroup/kogito-cloud-operator/pull/475/commits/b0cf158a0b11b4cae7c87f2ea6fd4ed3499db422)

File: docs/GUIDE_TO_KOGITO_DEVS.md (hunk 36 above)
Comment (Kaitou786) [reply docs/GUIDE_TO_KOGITO_DEVS.md]: The problem was because the readiness probe was failing for the `operator-catalog` pod causing none of the operator to be displayed.

**Comment by spolti** [comment]
is it ready to go?
btw, shouldn't we add this on the official doc as well?

**Comment by ricardozanini** [comment]
> is it ready to go?
> btw, shouldn't we add this on the official doc as well?

I believe we are ready, but let me ask one thing: this is more an internal doc for Kogito contributors/maintainers. I don't think we should add to the official docs at this moment. @sterobin let me know what you think.
